src/index.html

```diff
@@ -6,7 +6,37 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="calendar calendar--start-day--mon calendar--month-length--31">
```

Comment:
It's a bad practise add the modifiers to the body, you need to create a container for calendar days